### ISSUE TITLE
Split.io crashes app when launch without a connection

### ISSUE THREAD

**Issue by n8tr** [splitio/ios-client#54]
Split.io crashes from a force unwrap when app is launched when there is no internet connection (e.g. airplane mode, network connection is offline, or on cellular and data is turned off for a particular app)

line 39 in `HttpMySegmentsFetcher.swift`
```swift
        if !reachable {
            return (self.mySegmentCache?.getSegments(key: user))!

```
The user (key) is an empty string in this case, and the returned value is nil, which is then force-unwrapped.
 Simply removing the force-unwrap fixes the problem.

Recommendation: Never use force unwraps. Optionals in Swift, when used correctly, prevent crashes like this from ever happening.

**Comment by SenhorCastor** [comment]
Hi n8tr, 

Sorry for the inconveniences and thanks for flagging. Our team is addressing this. I will keep you posted with the status

**Comment by SenhorCastor** [comment]
HI n8tr,

To keep you up to date, we have the fix, we are running all our automation. I expect to publish a release today afternoon

**Comment by SenhorCastor** [comment]
Hi n8tr, we released 1.3.1 that addresses this issue.

Again sorry for the inconveniences, thanks for flagging and let us know if you need anything else.

**Comment by SenhorCastor** [comment]
Going to close for now.

Re-open or create a new issue if you have any other inconveniences
